File: array/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int answer = 0;
        int val = 0;
        for(int i = 0; i<nums.length; i++){
            if(nums[i] == 0){
                answer = Math.max(answer,val);
                val = 0;
            }else{
                val += 1;
            }
        }
        answer = Math.max(answer, val);
        return answer;
    }
}

File: array/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        OnesCount = 0
        maxCount = 0
        for ele in nums:
            if ele == 1:
                OnesCount +=1
            else:
                maxCount = max(maxCount,OnesCount)
                OnesCount = 0
        maxCount = max(maxCount,OnesCount)
        return maxCount
        

